### PR TITLE
[ADD] udes_stock: Transfer create procurement group on stock move 

### PR DESCRIPTION
**Pull request by armandcela**
SE-1442
Signed-off-by: Armand Cela <armand.cela@unipart.io>

### CODE REVIEW

File: addons/udes_stock/models/stock_move.py [hunk 1]
```diff
@@ -545,3 +545,29 @@ def _action_assign(self):
...
         super()._action_assign()
         return self
+
+    def _action_confirm(self, merge=True, merge_into=False):
+        """
+        Assign procurement group for moves where picking type create procurement groups is enabled
+        """
+        Procurement = self.env["procurement.group"]
+
+        for orderpoint_moves in self.filtered(
```
Comment (ws-mps) [inline addons/udes_stock/models/stock_move.py:555]: We can just loop over the moves that have `mv.picking_type_id.u_create_procurement_group and not mv.group_id` and then have an if to decide the name, then we only need 
```

Comment (armandcela) [reply addons/udes_stock/models/stock_move.py]: Pushed suggested changes

File: addons/udes_stock/models/procurement.py [hunk 2]
```diff
@@ -18,6 +18,17 @@ def _get_orderpoint_domain(self, company_id=False):
             domain += [('id', 'in', orderpoints)]
         return domain
 
+    @api.model
+    def _get_or_create_procurement_group(self, procurement_name):
```
Comment (ws-mps) [inline addons/udes_stock/models/procurement.py:22]: Can we use `mixin.stock.model` and then we will get get or create from there?

Comment (armandcela) [reply addons/udes_stock/models/procurement.py]: Pushed the suggested change

**Review by ws-mps — APPROVED** [review]

**Review by rajeeshknambiar — APPROVED** [review]

**Pull request merged**